src/redux/store.ts

```diff
@@ -56,12 +57,18 @@ export const makeStore = wrapMakeStore(() => {
     addressBookSlice.reducer
   );
 
+  const ensReducer = persistReducer(
+    { key: "ens-lockup", version: 1, storage: storageLocal },
+    ensApi.reducer
+  )
+
   const store = configureStore({
     reducer: {
       [rpcApi.reducerPath]: rpcApi.reducer,
       [sfSubgraph.reducerPath]: sfSubgraph.reducer,
       [themePreferenceSlice.name]: themePreferenceSlice.reducer,
-      [addressBookSlice.name]: addressBookReducer
+      [addressBookSlice.name]: addressBookReducer,
+      [ensApi.reducerPath]: ensReducer,
```

Comment:
I don't see much value in persisting ENS queries so we can just do `[ensApi.reducerPath]: ensApi.reducer,` here. 

This part can be removed:
```ts
  const ensReducer = persistReducer(
    { key: "ens-lockup", version: 1, storage: storageLocal },
    ensApi.reducer
  )
```